dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/NonTestDependencySelector.java

```diff
@@ -7,8 +23,9 @@
 /**
  * Selects dependencies except {@code test} scope.
  *
- * <p>{@link org.eclipse.aether.util.graph.selector.ScopeDependencySelector} has similar capability
- * but it selects all direct dependencies regardless of their scope.
+ * <p>This class selects all scopes except {@code test} even for direct dependencies, while Maven's
```

Comment:
all transitive dependencies that do not have scope test?

Reply:
This instance does not care about transitive or direct. Updated.